src/components/aside-bar/AsideBar.jsx

```diff
@@ -0,0 +1,71 @@
+import {
+  AiOutlineHome,
+  MdOutlineLabel,
+  BsArchive,
+  FiTrash,
+} from "assets/icons/icons";
+import { NavLink } from "react-router-dom";
+import { useNotesContext } from "context";
+import "./AsideBar.css";
+
+const AsideBar = () => {
+  const { notesDispatch } = useNotesContext();
+
+  return (
+    <aside className="aside-bar">
+      <ul className="ul-none aside-ul">
+        <NavLink
+          to="/notes"
+          className={({ isActive }) =>
+            isActive
+              ? "aside-link flex flex-start cursor active-link"
+              : "aside-link flex flex-start cursor"
```

Comment:
suggestion: you can use simple package called **clsx** that will reduce these two into one line